device/rsvp/rsvpInterface.yaml

```diff
@@ -0,0 +1,82 @@
+components:
+  schemas:
+    Rsvp.Interface:
+      x-status: under-review
+      description: >-
+        Configuration for RSVP Interface.
+      type: object
+      required: [ipv4_name,neighbor_ip]
+      properties:
+        ipv4_name: 
+          description: >-
+            The globally unique name of the IPv4 interface connected to the DUT.
+            This name must match the "name" field of the "ipv4_addresses" on top which this RSVP interface is configured.          
+          type: string
+          x-constraint:
+          - '/components/schemas/Device.Ipv4/properties/name'
+        neighbor_ip:
+          description: >-
+            IPv4 address of the RSVP neighbor on this interface.
+          type: string
+          format: ipv4
+        label_space_start:
+          description: >-
+            The user-defined label space start value. The LSPs for which this router acts as a egress are assigned labels from this
+            label pool.The"label_space_start" and "label_space_end" together defines this label-pool. 
+          type: integer
+          default: 1000
+          minimum: 0
+          maximum: 1048575
+        label_space_end:
+          description: >-
+            The user-defined label space end value.The last label value that can be assigned to the LSPs for which this router acts as egress.
+          type: integer
+          default: 100000
```

Comment:
It is just a reasonably large value which allows support of ~ 99000 lsps per RSVP router .  We normally dont see a need to initiate higher number of LSPs.  But the control of the min and max of the label space is more important than the values themselves so that labels provided by other label providers dont clash with RSVP values .